### PR TITLE
fix two issues with virtual environment

### PR DESCRIPTION
**Pull request by hhbyyh**
I went through the doc steps for running with virtual environment and found two issues:
1. the python_package.sh can only be executed when current directory is `bin` folder, otherwise it cannot find requirement.txt

2. the scripts for running with virtual environment ran with error:

`Caused by: java.io.IOException: Cannot run program "/root/dist/bin/venv.zip/venv/bin/python": error=2, No such file or directory`

The PR tries to fix both issues.

### CODE REVIEW

**Comment by hkvision** [comment]
Emmm... this guide originally comes from https://github.com/intel-analytics/analytics-zoo/tree/master/scripts#run-scripts

**Comment by hhbyyh** [comment]
Thanks @hkvision so we need to update both doc and readme?

**Comment by hkvision** [comment]
Wondering can we use our script `spark-submit-with-zoo` for yarn so that users don't need to specify the jars and python zips? I find the document site out of date as the jar names are wrong...

**Comment by hkvision** [comment]
Thanks for testing and reporting this. Let's discuss tomorrow and I will help the test. Let's try to fix is asap.

**Comment by hhbyyh** [comment]
Yes I'm using submit script and it's working.

**Comment by hkvision** [comment]
- Then can we update the command with script?
- In the readme can we just link to the website page so that we don't need to maintain two places of instructions?
- Maybe can we add/update `export ANALYTICS_ZOO_HOME=the dist directory under the Analytics Zoo project` here: https://analytics-zoo.github.io/master/#PythonUserGuide/install/#for-yarn-cluster

File: docs/docs/PythonUserGuide/run.md [hunk 1]
```diff
@@ -138,54 +138,34 @@ Here we use Analytics Zoo [Object Detection Python example](https://github.com/i
 * Yarn cluster mode
 ```
     SPARK_HOME=the root directory of Spark
```
Comment (hkvision) [inline docs/docs/PythonUserGuide/run.md]: If we use script, we need to add `export` to SPARK_HOME and `ANALYTICS_ZOO_HOME`

File: docs/docs/PythonUserGuide/run.md [hunk 2]
```diff
@@ -138,54 +138,34 @@ Here we use Analytics Zoo [Object Detection Python example](https://github.com/i
...
     --driver-memory 10g \
     --executor-cores 8 \
     --num-executors 2 \
-    --properties-file ${ANALYTICS_ZOO_CONF} \
-    --jars ${ANALYTICS_ZOO_JAR} \
-    --py-files ${ANALYTICS_ZOO_PY_ZIP} \
     --archives ${VENV_HOME}/venv.zip \
-    --conf spark.driver.extraClassPath=${ANALYTICS_ZOO_JAR} \
-    --conf spark.executor.extraClassPath=${ANALYTICS_ZOO_JAR} \
     ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/objectdetection/predict.py model_path image_path output_path
```
Comment (hkvision) [inline docs/docs/PythonUserGuide/run.md]: change this just to `predict.py model_path image_path output_path`? as now there's no ANALYTICS_ZOO_ROOT

File: docs/docs/PythonUserGuide/run.md [hunk 3]
```diff
@@ -138,54 +138,34 @@ Here we use Analytics Zoo [Object Detection Python example](https://github.com/i
...
 ```
 
 * Yarn client mode
 ```
     SPARK_HOME=the root directory of Spark
```
Comment (hkvision) [inline docs/docs/PythonUserGuide/run.md]: same as above

File: docs/docs/PythonUserGuide/run.md [hunk 4]
```diff
@@ -138,54 +138,34 @@ Here we use Analytics Zoo [Object Detection Python example](https://github.com/i
...
     --driver-memory 10g \
     --executor-cores 16 \
     --num-executors 2 \
-    --properties-file ${ANALYTICS_ZOO_CONF} \
-    --jars ${ANALYTICS_ZOO_JAR} \
-    --py-files ${ANALYTICS_ZOO_PY_ZIP} \
     --archives ${VENV_HOME}/venv.zip \
-    --conf spark.driver.extraClassPath=${ANALYTICS_ZOO_JAR} \
-    --conf spark.executor.extraClassPath=${ANALYTICS_ZOO_JAR} \
     ${ANALYTICS_ZOO_ROOT}/pyzoo/zoo/examples/objectdetection/predict.py model_path image_path output_path
```
Comment (hkvision) [inline docs/docs/PythonUserGuide/run.md]: same as above

**Comment by hkvision** [comment]
@hhbyyh Feel free to merge this if OK. And it is better if we backport to branch 0.1/0.2/0.3 = =

**Comment by hkvision** [comment]
I have finished the backport.

**Comment by hhbyyh** [comment]
Thanks @hkvision .

**Pull request merged**